ukat/mapping/t1.py

```diff
@@ -67,6 +68,9 @@ def __init__(self, pixel_array, inversion_list, affine, tss=0, tss_axis=-2,
             The number of parameters to fit the data to. A two parameter fit
             will estimate S0 and T1 while a three parameter fit will also
             estimate the inversion efficiency.
+        molli : bool
```

Comment:
Maybe add "bool, optional" since it's a keyword argument

Reply:
Done